includes/class-webmention-receiver.php

```diff
@@ -210,6 +222,41 @@ public static function get( $request ) {
 		return true;
 	}
 
+	/**
+	 * Store extra parameters in comment meta
+	 *
+	 * @param array $commentdata
+	 *
+	 * @return array|null|WP_Error $commentdata The Comment Array with added comment meta field or a WP_Error object.
+	 */
+	public static function store_meta( $commentdata ) {
```

Comment:
why not directly set them to `$commentdata['comment_meta']`? would save us a lot of code.

Reply:
I wanted to separate out the comment meta storage from the comment data storage. Mixing them was somewhat confusing.

Reply:
I can try to reorder it differently.

Reply:
I am going to split off the non controversial parts of this and take a separate shot at the other piece